sdk/eventhub/event-hubs/src/managementClient.ts

```diff
@@ -296,6 +296,12 @@ export class ManagementClient extends LinkEntity {
         maxRetries: options.retryOptions && options.retryOptions.maxRetries,
         abortSignal: options.abortSignal,
         requestName: options.requestName,
+        timeoutInSeconds:
+          options.retryOptions == undefined ||
+          options.retryOptions.operationTimeoutInMs == undefined ||
+          options.retryOptions.operationTimeoutInMs < 0
+            ? Constants.defaultOperationTimeoutInSeconds
+            : options.retryOptions.operationTimeoutInMs / 1000,
```

Comment:
We had feedback that we should have a minimum value for per operation timeout, else it is easy to abuse the system by forcing frequent timeouts by giving smaller values.

Use the `Constants.defaultOperationTimeoutInSeconds` as the "minimum" for now

Reply:
I see that https://github.com/Azure/azure-sdk-for-js/pull/4239/commits/4ba2e1437d35db5927610a083a4060cc6c9dec5b commit has been made in an attempt to address the issue, but the change is not solving the problem at hand.

Use `Constants.defaultOperationTimeoutInSeconds` as minimum value means that if user provides a value lesser than this, then we ignore user's value and use `Constants.defaultOperationTimeoutInSeconds` instead

Reply:
Updated